src/main.rs

```diff
@@ -350,7 +350,7 @@ fn main() {
 
     channels_wrapper.add(&app.channels);
 
-    let add = Button::new_with_label("Add...");
+    let add = Button::new_with_mnemonic("Add _Channel");
```

Comment:
Doesn't this cause an issue with `Edit message`'s Cancel button?

Reply:
No, tested it

Reply:
Awesome, thanks!